### PR TITLE
CONSOLE-4126: replace deprecated components in OperatorChannelSelect …

### PR DESCRIPTION
**Pull request by rhamilto**
…and OperatorVersionSelect

After

https://github.com/user-attachments/assets/c32223f2-ae80-4773-83f8-187c03f8885d

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-channel-version-select.tsx [hunk 1]
```diff
@@ -41,25 +36,33 @@ export const OperatorChannelSelect: React.FC<OperatorChannelSelectProps> = ({
...
+          setUpdateVersion('');
+        }}
+        selected={selectedUpdateChannel || '-'}
+        onOpenChange={(isOpen) => setIsChannelSelectOpen(isOpen)}
         isOpen={isChannelSelectOpen}
-        selections={selectedUpdateChannel || '-'}
-        onSelect={handleChannelSelection}
-        isDisabled={!packageManifest}
       >
         {channelSelectOptions}
```
Comment (rhamilto) [inline frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-channel-version-select.tsx:64]: Missing `<SelectList>`

File: frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-channel-version-select.tsx [hunk 2]
```diff
@@ -122,17 +119,30 @@ export const OperatorVersionSelect: React.FC<OperatorVersionSelectProps> = ({
...
+          setIsVersionSelectOpen(false);
+        }}
+        selected={selectedUpdateVersion}
+        onOpenChange={(isOpen) => setIsVersionSelectOpen(isOpen)}
         isOpen={isVersionSelectOpen}
-        selections={selectedUpdateVersion}
-        onSelect={handleVersionSelection}
-        isDisabled={!packageManifest}
       >
         {versionSelectOptions}
```
Comment (rhamilto) [inline frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-channel-version-select.tsx:144]: Missing `<SelectList>`

**Comment by rhamilto** [comment]
Tech debt so manually adding labels:
/label px-approved
/label docs-approved

/assign @yapei
/assign @sg00dwin

**Review by sg00dwin — APPROVED** [review]
/lgtm

**Comment by yapei** [comment]
Deprecated icon can not be shown in `Channel` and `Version` dropdown
Screenshot 2024-08-21 at 4 06 50 PM

**Comment by rhamilto** [comment]
> Deprecated icon can not be shown in `Channel` and `Version` dropdown Screenshot 2024-08-21 at 4 06 50 PM

Great catch, @yapei!

Updated

https://github.com/user-attachments/assets/1123f252-4f99-44b8-968e-efc8536dfbbc

@cyril-ui-developer, PTAL

**Pull request merged**